### ISSUE TITLE
How about video embed with thumbnail image?

### ISSUE THREAD

**Issue by bsdo64** [orthes/medium-editor-insert-plugin#261]
In my app,
embeding video is too slow when rendering because calling some scripts.

<img width="1300" alt="2016-01-11 3 52 24" src="https://cloud.githubusercontent.com/assets/5698928/12228213/662e4402-b87b-11e5-9c11-f24e0a30578a.png">

How about changes embed video by thumbnails?
http://www.sitepoint.com/faster-youtube-embeds-javascript/

**Comment by anilmaurya** [comment]
I am also using medium-insert-plugin and embedding video is not slow for me.

My console log look like this 

![screen shot 2016-01-21 at 3 30 57 pm](https://cloud.githubusercontent.com/assets/1912864/12477201/37f246ca-c054-11e5-9f21-d7c5ade7c43e.png)

In your console log, I see every script is called 7 times, may be you are binding events 7 times (unknowingly) . May be this is the reason embedding is slow for you.

**Comment by bsdo64** [comment]
I have post list in one page app. In there there are more than 10 posts at one request (infinite scroll) Do you also request just 7 times while calling 10 videos?

**Comment by anilmaurya** [comment]
I am not showing embedded videos on list page, instead I show thumbnail image on list page.

I show embedded video on post show page and editor.

Check this https://followme.co/anilmaurya2

**Comment by anilmaurya** [comment]
Listing 10 embedded video on single page will definitely slow page loading.

I implemented http://www.sitepoint.com/faster-youtube-embeds-javascript/ in earlier version of my editor, its not tough to implement and is light weight too.

+1 for this.

**Comment by bsdo64** [comment]
Um... I'm showing posts list with all contents like facebook. I think I have to parse contents and change embeds to thumbnail image manually. Thank you anyway

**Comment by anilmaurya** [comment]
@j0k3r @orthes I can work on this and send a pull request. Let me know if you are interested in this feature.

**Comment by orthes** [comment]
@anilmaurya Yeah, if you have it, it would be great if you could submit a PR.

**Comment by anilmaurya** [comment]
@orthes I will work on it and send pull request in couple of days.

**Comment by jcalfee** [comment]
If it helps, we just implemented this in our viewer using the "Faster YouTube Embeds with JavaScript" example..  This is the initial react component.  The css is the same (from the example).

https://github.com/steemit/steemit.com/blob/43e32326e3405cbc7bc26898fab848ef4cf4b264/app/components/elements/YoutubePreview.jsx
